### PR TITLE
Add portmgr entry for 2022q2

### PR DESCRIPTION
**Pull request by rene0**

### CODE REVIEW

File: 2022q2/portmgr.adoc [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+At the time of writing, there are 2,700 open ports PRs of which 682 are unassigned.
+Compared to the previous quarter, the Ports Tree grew by 7%, a slight decrease in commit activity and a constant number of PRs.
+
+During the last quarter, portmgr
+welcomed back salvadore@ but also said goodbye to seven ports committers due to lack of activity.
+
+In its bi-weekly meetings, portmgr discussed the following topics:
+* the future of ca_root_nss
+* feasibility of the base system providing certain .pc files
+* ways to deal with incompatibilities kernel module ports on minor version upgrades of the base system
```
Comment (pauamma) [inline 2022q2/portmgr.adoc:30]: "incompatibilities kernel module ports" Missing "in" here, or perhaps "between".

File: 2022q2/portmgr.adoc [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+Following a discussion among developers, portmgr decided to grant all documentation and source
+committers approval to fix any documentation-related change in Ports Tree which does not affect its functionality.
```
Comment (pauamma) [inline 2022q2/portmgr.adoc:33]: "in the Ports Tree"

Also, either "fix any documentation-related error" or "make any documentation-related change"

**Review by pauamma — CHANGES_REQUESTED** [review]

File: 2022q2/portmgr.adoc (hunk 1 above)
Comment (rene0) [reply 2022q2/portmgr.adoc]: Fixed

File: 2022q2/portmgr.adoc (hunk 2 above)
Comment (rene0) [reply 2022q2/portmgr.adoc]: Also fixed, thanks for the review.

**Review by pauamma — APPROVED** [review]
Comment can be removed on or after merging.

**Comment by rene0** [comment]
Feel free to commit, I don't have write access here iirc.

**Pull request merged**